### DISCUSSION TITLE
Welcome to GPRPy Discussions!

### DISCUSSION THREAD

**Discussion by AlainPlattner** [NSGeophysics/GPRPy#23] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sunningple123** [comment]
I am doing a GPR data processing. But the data's format is not gpr or others supported. I only have traces matrix. I wonder what shall I do to process data using GPRPy?
